zerver/openapi/zulip.yaml

```diff
@@ -14519,6 +14519,15 @@ paths:
         - Stream [permissions](/help/stream-permissions), including
           [privacy](/help/change-the-privacy-of-a-stream) and [who can
           send](/help/stream-sending-policy).
+
```

Comment:
The above paragraph does not mention about `message_retention_days` which is also updated using this endpoint. I am not sure we would consider it as a permission. So, can we just write stream settings and inlcude all 3 or keep them stream permissions and mention `message_retention_days` as a heading. This might update the below line as well.

Reply:
I think probably "stream settings" would be fine, but maybe including `message_retention_days` as an example in the "including" list would be good.